library/datetime/now_spec.rb

```diff
@@ -5,4 +5,25 @@
   it "creates an instance of DateTime" do
     DateTime.now.should be_an_instance_of(DateTime)
   end
+
+  it "sets the current date" do
+    dt = DateTime.now
+    date = Date.today
+    dt.year.should == date.year
+    dt.mon.should == date.mon
+    dt.day.should == date.day
+  end
+
+  it "sets the current time" do
+    dt = DateTime.now
+    now = Time.now
+    (dt.to_time - now).should be_close(24 * 60 * 60)
```

Comment:
This can be more precise, like `be_close(10.0)` since they both have at least second precision.